engine/installation/linux/oracle.md

```diff
@@ -210,7 +274,6 @@ To upgrade Docker, download the newer package file and repeat the
 [installation procedure](#install-from-a-package), using `yum -y upgrade`
```

Comment:
Why are you advocating the use of the `-y` parameter? This would prevent the user from confirming the install. It is usually only recommended when triggering `yum` from an automation or management system.